introduction/key-concepts-in-glaredb.md

```diff
@@ -1,20 +1,149 @@
 ---
-title: Key Concepts in GlareDB
+title: Key Concepts
 layout: default
 nav_order: 1
 parent: Getting Started
 ---
 
-## Key Concepts in GlareDB
+# Key Concepts in GlareDB
 
-### Table Functions
+## Table Functions
 
-### External Data Sources
+A core tenet of GlareDB is that [All Data are SQL Addressable], even data not
+in a SQL database.
 
-### Native Data Storage
+Table functions access data, on-demand, wherever it is, including data sources
+that have not been added to GlareDB.
```

Comment:
"not added to glaredb" is sort of confusing? I think you mean "registered as an external table" but it makes it sound like an implementation deficiency